### ISSUE TITLE
Devnet readiness

### ISSUE THREAD

**Issue by bulenttastan** [MovedNetwork/op-move#180]
Tools and infra to get ready for the Devnet launch.

Some tasks can be big and some very easy. This issue is to make sure at least the initial steps are ready for developers to start building and users to start bridging tokens to use the chain.

- [x] Set up a faucet web page to load up L1 tokens
- [x] Infra to host L1 Geth and L2 RPC endpoints
- [x] Documentation on how a developer can get started
- [x] Install an explorer to trace transactions

**Comment by birchmd** [comment]
> Determine gas costs for EVM contract and native function calls

I don't think this is a blocker for devnet. We will need to get this sorted out before testnet, but I think using devnet as an opportunity to learn more about real costs of the execution would be helpful in estimating the gas costs.

**Comment by bulenttastan** [comment]
Faucet is online at: https://moved.network/faucet

Ethereum Devnet node is running at: https://ethereum.moved.network

**Comment by bulenttastan** [comment]
Moved Devnet: https://devnet.moved.network

Block Explorer: https://devnet.explorer.moved.network
